### DISCUSSION TITLE
lib/forge-std not installing with forge init

### DISCUSSION THREAD

**Discussion by Avnish-07** [Cyfrin/foundry-full-course-cu#1663] [Q&A] [answered]
when I'm initializing it by forge init the "lib/forge-std" is not installing
![Screenshot 2024-04-30 071910](https://github.com/Cyfrin/foundry-full-course-f23/assets/133528639/40774ca2-754b-464c-8f8c-fad13f88239b)

**Comment by EngrPips** [comment, marked as answer]
You can install it manually using the command below.

```javascript
forge install foundry-rs/forge-std --no-commit
```

**Reply by Avnish-07** [reply]
thank you

**Reply by EngrPips** [reply]
Anytime, friend, Keep crushing it.
